pages/docs/reference/js-modules.md

```diff
@@ -19,68 +19,77 @@ the list of available options:
 4. Unified Module Definitions (UMD), which is compatible with both *AMD* and *CommonJS*, and works as "plain"
    when neither *AMD* nor *CommonJS* is available at runtime.
 
+## Targeting the browser
+ 
+If you're targeting the browser, you can specify the desired module type in the `webpackTask` configuration block:
+ 
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
+
+```groovy
+kotlin {
+    target {
+        browser {
+            webpackTask {
+                output.libraryTarget = COMMONJS 
```

Comment:
I think it is useful to provide information what this declaration is

I mean impoty
```
import org.jetbrains.kotlin.gradle.targets.js.webpack.KotlinWebpackOutput.Target.COMMONJS
...
kotlin {
    target {
        browser {
            webpackTask {
                output.libraryTarget = COMMONJS 
```
So need to explicit import for understandind where other variants can be found